pkg/restapi/issuer/operation/operations.go

```diff
@@ -505,7 +499,14 @@ func (o *Operation) retrieveCredentialHandler(rw http.ResponseWriter, req *http.
 		}
 	}
 
-	o.retrieveCredential(rw, profile, docURLs)
+	o.retrieveCredential(rw, profileName, profile.EDVVaultID, docURLs)
+}
+
+func (o *Operation) getVaultIDFromVaultLocationURL(vaultLocationURL string) string {
```

Comment:
since this function is not accessing any elements from `Operation` type, you can move it out.

Reply:
can be shortened `return vaultLocationURLSplitUp[len(vaultLocationURLSplitUp)-1]`

Reply:
fixed!